calculate_average.js

```diff
@@ -0,0 +1,14 @@
+
+const calculateAverage = function calculateAverage(grades) {
```

Comment:
Can you tell me more why this function is assigned to a constant of the same name?